src/metrics.rs

```diff
@@ -451,6 +458,26 @@ impl GooseRequestMetricAggregate {
         self.status_code_counts.insert(status_code, counter);
         debug!("incremented {} counter: {}", status_code, counter);
     }
+
+    /// Record request in requests per second metric.
+    pub(crate) fn record_requests_per_second(&mut self, bucket_size: usize) {
+        let time_bucket = (Utc::now().timestamp() - self.start_time) as usize / bucket_size;
```

Comment:
Instead of the actual time of the request we are using the time when the request metric is reported. 

It would be better to collect the time when the request happened, but I think that this data currently isn't reported by the worker.